space-time.Rmd

```diff
@@ -32,32 +34,40 @@ ggplot(small, aes(eruptions, waiting)) +
 
 For interactive 3d plots, including true 3d surfaces, see RGL, <http://rgl.neoscientists.org/about.shtml>.
 
-## Drawing maps {#maps}
+## Vector maps {#maps}
```

Comment:
This immediately made me think of `geom_arrow()` (i.e. vector fields) — I wonder if we should mention that in this chapter too?